### PR TITLE
Assorted improvements to the gem's dependencies.

### PR DESCRIPTION
**Pull request by lucasmazza**

### CODE REVIEW

**Comment by jonrohan** [comment]
Awesome thanks for the contribution @lucasmazza! Gonna take me a couple of days to review this.

File: Gemfile [hunk 1]
```diff
@@ -2,9 +2,6 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'octicons', '~> 2.0'
```
Comment (jonrohan) [inline Gemfile]: Why remove octicons? It's a requirement for this to work.

**Comment by jonrohan** [comment]
I can't get this to run locally 

```
❯ bin/bundle update 
Fetching https://github.com/lucasmazza/octicons_helper.git
Fetching gem metadata from https://rubygems.org/.........
Fetching version metadata from https://rubygems.org/...
Fetching dependency metadata from https://rubygems.org/..
Resolving dependencies....................................................................................
Bundler could not find compatible versions for gem "activesupport":
  In Gemfile:
    octicons_helper was resolved to 2.1.0, which depends on
      actionview was resolved to 4.1.0, which depends on
        activesupport (= 4.1.0)
```

Happens when I bundle in my project

File: Gemfile (hunk 1 above)
Comment (lucasmazza) [inline Gemfile]: The dependency is already defined inside the gemspec, and we are loading these dependencies through the `gemspec` method call on line 3 here.

**Comment by lucasmazza** [comment]
@jonrohan that's weird - can you share more of the `bundle update` output? The following lines would have more details regarding the dependency mismatch. Later today I'll try to install the gem on my Rails app to compare the results.

**Comment by lucasmazza** [comment]
@jonrohan I managed to install the gem on my current app (Rails 5) as expected:

``` diff
diff --git a/Gemfile b/Gemfile
index abe2e6e..200e0fc 100644
--- a/Gemfile
+++ b/Gemfile
@@ -82,3 +82,5 @@
+gem 'octicons_helper', git: 'https://github.com/lucasmazza/octicons_helper.git', branch: 'lm-deps-fix'
diff --git a/Gemfile.lock b/Gemfile.lock
index 7fefded..171143a 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -21,6 +21,16 @@ GIT
       tty-spinner (~> 0.1.0)

 GIT
+  remote: https://github.com/lucasmazza/octicons_helper.git
+  revision: cee6577828142640f0a563cfcae5a683ae0ff1e7
+  branch: lm-deps-fix
+  specs:
+    octicons_helper (2.1.0)
+      actionview
+      octicons (~> 2.0)
+      railties
+
+GIT
```

**Comment by jonrohan** [comment]
So I think the problem might be that, I need to support an older version of rails

```
❯ bin/rails -v
Rails 3.2.22.4
```

The way you have cleaned up the gem might only work in Rails 4< ?

**Comment by lucasmazza** [comment]
> The way you have cleaned up the gem might only work in Rails 4< ?

Yeah, `actionview` started with Rails 4.1.0 :(. But we can depend on `actionpack` instead, which is available for Rails 3. I'll test it out with a Rails 3.2 app and report here later.

**Comment by jonrohan** [comment]
closing in favor of https://github.com/primer/octicons_helper/pull/3

**Pull request closed without merging**